apps/dispatch/src/app/components/simulations/view/view.service.ts

```diff
@@ -84,14 +87,14 @@ export class ViewService {
     );
     return {
       id: simulation.id,
-      name: simulation.name,
-      simulator: simulation.simulator,
-      simulatorVersion: simulation.simulatorVersion,
-      cpus: simulation.cpus || 1,
-      memory: simulation.memory || 8,
-      maxTime: simulation.maxTime || 20,
-      envVars: simulation.envVars || [],
-      status: simulation.status,
+      name: simulation?.name as string,
```

Comment:
@jonrkarr. This effectively breaks type checking as type assertions override the compiler. I am not sure If this is the best approach to use here. Anything downstream of this method now needs to manually ensure that the data is actually a string and not null or undefined.

Reply:
I realize this. That class is being used for two things. More work is needed to better separate this out.